Sources/TuistKit/Commands/DecryptCommand.swift

```diff
@@ -0,0 +1,48 @@
+import Basic
+import Foundation
+import SPMUtility
+import TuistCore
+import TuistSigning
+import TuistSupport
+
+class DecryptCommand: NSObject, Command {
+    // MARK: - Attributes
+
+    static let command = "decrypt"
+    static let overview = "Decrypts all files in Tuist/Signing directory."
+    private let pathArgument: OptionArgument<String>
+
+    private let signingCipher: SigningCiphering
+
+    // MARK: - Init
+
+    public required convenience init(parser: ArgumentParser) {
+        self.init(parser: parser, signingCipher: SigningCipher())
+    }
+
+    init(parser: ArgumentParser,
+         signingCipher: SigningCiphering) {
+        let subParser = parser.add(subparser: DecryptCommand.command, overview: DecryptCommand.overview)
+        pathArgument = subParser.add(option: "--path",
+                                     shortName: "-p",
+                                     kind: String.self,
+                                     usage: "The path to the folder where the template will be generated (Default: Current directory).",
```

Comment:
This usage text is incorrect, how about:

> The path to the folder containing the encrypted certificates

Reply:
Copy & pasting programming :trollface:

Reply:
Busted 🚓 👮